UPGRADING.md

```diff
@@ -109,6 +109,18 @@ will need to change to accommodate these changes. Most notably:
   longer exported and have been replaced with `node.New` and
   `node.NewDefault` which provide more functional interfaces.
 
+To access any of the functionality previously avalible vai the
+`node.Node` type, use the `*local.Local` "RPC" client, that exposes
+the full RPC interface provided as direct function calls. Import the
+`github.com/tendermint/tendermint/rpc/client/local` package and pass
+the node service as in the following: 
+
+    node := node.NewDefault() //construct the node object
+    // start and set up the node service 
+
+	client := local.New(node.(local.NodeService))
+	// use client object to interact with the node
```

Comment:
```suggestion
```go
    node := node.NewDefault() //construct the node object
    // start and set up the node service 

    client := local.New(node.(local.NodeService))
    // use client object to interact with the node
```
```

Reply:
Lol I can't add the code blocks. Inception doesn't work with markdown

Reply:
The main point of this wasn't so much the code blocks (although I think they look better) but more that the indentation was the same

Reply:
oh nice.

what is inception?

Reply:
Well recursion really. I was creating a code block for the suggestion that had a code block for the go code inside it. (Which turns out doesn't work)